### PR TITLE
Initialize vaadin-avatar

### LINKED ISSUE #1: Setup vaadin-avatar project
Init as Polymer 2 project.
Setup CI (Travis setup + Sauce access).
Setup GitHub (branch protection at least)

### PR DESCRIPTION
**Pull request by yuriy-fix**
Fixes #1

### CODE REVIEW

File: .gemini.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  chrome:
+    desiredCapabilities:
+      browserName: "chrome"
+      version: "66.0"
+      platform: "Windows 10"
+
+  firefox:
+    desiredCapabilities:
+      browserName: "firefox"
+      version: "47.0"
```
Comment (web-padawan) [inline .gemini.yml]: IIRC we agreed to remove Firefox, as it's quite outdated. So this part can be removed. 

Not a blocker.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    - if: type = push
+      env: TEST_SUITE=unit_tests POLYMER=2
+    - if: type = push
+      env: TEST_SUITE=visual_tests POLYMER=2
+    - if: type = push
+      env: TEST_SUITE=unit_tests POLYMER=3
+    - if: type = pull_request
+      env: POLYMER=2
+      addons:
+        firefox: "66.0"
```
Comment (web-padawan) [inline .travis.yml]: You can use `latest`, the problem that forced us to pin this version has been fixed.

File: LICENSE [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+      License. However, in accepting such obligations, You may act only
+      on Your own behalf and on Your sole responsibility, not on behalf
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
+
+   Copyright 2019 Vaadin Ltd.
```
Comment (web-padawan) [inline LICENSE]: nit: 2020

File: src/vaadin-avatar.html [hunk 4]
```diff
@@ -0,0 +1,62 @@
+<!--
+@license
+Copyright (c) 2019 Vaadin Ltd.
```
Comment (web-padawan) [inline src/vaadin-avatar.html]: nit: 2020

File: .gemini.yml (hunk 1 above)
Comment (yuriy-fix) [reply .gemini.yml]: Updated.

File: .travis.yml (hunk 2 above)
Comment (yuriy-fix) [reply .travis.yml]: Updated.

File: LICENSE (hunk 3 above)
Comment (yuriy-fix) [reply LICENSE]: Updated.

File: src/vaadin-avatar.html (hunk 4 above)
Comment (yuriy-fix) [reply src/vaadin-avatar.html]: Updated.

**Review by web-padawan — APPROVED** [review]

**Pull request merged**